projects/uitdatabank/docs/entry-api/events/create-uitpas.md

```diff
@@ -1,10 +1,275 @@
+
+To include the UiTPAS prices of an event when fetching it, you must add an `embedUitpasPrices` query parameter to the URL and set it to `true`.
+
+For example:
+
+```http
+GET /event/a4425f9a-bc3a-4a45-ab20-a23b58e8259c?embedUitpasPrices=true HTTP/1.1
+Host: https://io.uitdatabank.be
+```
+
+In addition to the price tariffs stored by your integration, one or more tariffs with the category `uitpas` will be included in the event's `priceInfo` property:
+
+```json
+{
+  "priceInfo": [
+    {
+      "category": "base",
+      "name": {
+        "nl": "Basistarief",
+        "fr": "Tarif de base",
+        "en": "Base tariff",
+        "de": "Basisrate"
+      },
+      "price": 3.1,
+      "priceCurrency": "EUR"
+    },
+    {
+      "category": "uitpas",
+      "name": {
+        "nl": "Kansentarief met UiTPAS Pelt"
+      },
+      "price": 0.62,
+      "priceCurrency": "EUR"
+    }
+  ]
+}
+```
+
+Try it now:
+
+```json http
+{
+  "url": "https://io.uitdatabank.be/event/a4425f9a-bc3a-4a45-ab20-a23b58e8259c",
+  "method": "get",
+  "query": {
+    "embedUitpasPrices": "true"
+  }
+}
+```
+
+<!-- theme: warning -->
+
+> **The `embedUitpasPrices` parameter has no performance impact.** It only serves as a temporary feature toggle.
+>
+> The default value of the `embedUitpasPrices` parameter is planned to change to `true` in the future.
+>
+> It is set to `false` for the time being so that existing integrations can make any changes necessary before the UiTPAS prices are included by default. For example, so they can remove their existing integration with UiTPAS API to fetch the prices because this would otherwise result in duplicate UiTPAS prices being shown.
+>
+> If you do not wish to ever display UiTPAS prices, make sure to explicitly set the `embedUitpasPrices` parameter to `false` to exclude them when the default changes to `true` in the future.
+
+## Changing the card systems of an UiTPAS event
+
+By default, all the card systems that the event's UiTPAS organizer is active in will be enabled. This means that passholders from those card systems will be able to buy tickets with an UiTPAS discount.
+
+For example: If the event's organizer is active in both UiTPAS Gent and UiTPAS Oostende, those card systems will be enabled by default on the event and passholders from both regions will be able to buy tickets with an UiTPAS discount for the event.
+
+In the near future you will be able to enable/disable specific card systems on your UiTPAS events, so you can specify for which ones discounts should be provided. **This functionality is still under construction, please check back soon.** <!-- @todo Link to an UiTPAS guide on how to configure cardSystems on the event when that is ready. -->
+
+When you enable/disable card systems on your UiTPAS event, the event's [UiTPAS labels](#uitpas-labels) will also be updated to reflect the enabled card systems. UiTPAS labels of disabled card systems are removed from the event.
```

Comment:
I thought this was already in production. 
https://docs.publiq.be/docs/uitpas/uitpas-api/reference/operations/update-a-event-card-system

Reply:
Yes but not the changes that we discussed in the meeting with Corneel/Erwin/Jean-Marie (and you and me), like the permission check that needs to be updated. Entry API integrators are not able to use it until that work is done